registration/models.py

```diff
@@ -11,7 +20,12 @@ class Family(models.Model):
         validators=[validate_family_parent],
     )
     email = models.EmailField(blank=True)
-    phone_number = models.CharField(max_length=128, blank=True)
+    home_number = models.CharField(max_length=128, blank=True, default="")
+    cell_number = models.CharField(max_length=128, blank=True, default="")
+    work_number = models.CharField(max_length=128, blank=True, default="")
```

Comment:
Should we enforce one of these to be mandatory?

Reply:
I think we can leave it flexible for the staff :)